src/etc/inc/interfaces.inc

```diff
@@ -2641,6 +2652,7 @@ function interface_configure($interface = 'wan', $reloadall = false, $linkupeven
                 interface_6to4_configure($interface, $wancfg);
                 break;
             case 'track6':
+				configure_dhcp6c_conf('wan', $config['interfaces']['wan']);
```

Comment:
And here we need to look up the track interface instead of guessing WAN, right?

Reply:
I guess we could, but at this point I am very wary of a user trying to use two WAN dhcp6 interfaces, that opens up a whole new can of worms I am yet to look at. My reading of dhcp6c says it's possible, and one of the reasons for this approach. I just have gone there .... yet. :)

Reply:
the problem is: people may add a second wan (e.g. opt1) and delete wan... so we come up empty here

Reply:
here we'll have to rewrite the interface that is being tracked

Reply:
Not only that, I have also realised that if its changed away from track-interface, then the config needs to reflect that. I am currently working on that, but I'll hold until you have finished whatever else you are doing.